multi-node/aws/pkg/config/config.go

```diff
@@ -126,6 +127,16 @@ func (c Cluster) Config() (*Config, error) {
 	config.APIServers = fmt.Sprintf("http://%s:8080", c.ControllerIP)
 	config.SecureAPIServers = fmt.Sprintf("https://%s:443", c.ControllerIP)
 	config.APIServerEndpoint = fmt.Sprintf("https://%s", c.ExternalDNSName)
+	if config.UseCalico {
+		config.K8sNetworkPlugin = "cni"
+		config.CalicoEnable = "true"
+		// Use a Hyperkube with CNI binaries
+		config.K8sVer = config.K8sVer + "-cni"
+	}else{
+		config.CalicoEnable = "false"
```

Comment:
It'll work! Golang boolean serialization is cloud-config compatible!

https://play.golang.org/p/xefEIUcA1M